### ISSUE TITLE
Cannot import CEL if certain letters appear in strings

### ISSUE THREAD

**Issue by ozgurcanturna** [react-querybuilder/react-querybuilder#358]
If we have to a text that includes some keywords like "and" or "in" parsing failed in CEL Import. 

This problem can be reproduced on [https://react-querybuilder.js.org/react-querybuilder](https://react-querybuilder.js.org/react-querybuilder ) demo page. if you select Load From CEL and give the query as;

firstName.startsWith("Stin") && age > 28

you can see the parsing failed . 

Now we try to replace the text with some other like( instead of  "in" >  "__i__" )  but this is not the correct solution. The parser should not look in String fields for keywords.

**Comment by jakeboone02** [comment]
This is actually worse than you guessed! Turns out it fails if the string has "r" or "n" in it. See [this codesandbox](https://codesandbox.io/s/tender-shockley-fr8n0b?file=/src/App.tsx). Code:

```tsx
export default function App() {
  return (
    <div style={{ whiteSpace: "nowrap" }}>
      {"abcdefghijklmnopqrstuvwxyz".split("").map((letter) => (
        <div key={letter}>
          {letter}: {JSON.stringify(parseCEL(`f1.startsWith("${letter}")`))}
        </div>
      ))}
    </div>
  );
}
```

Result:

```
a: {"rules":[{"field":"f1","operator":"beginsWith","value":"a"}],"combinator":"and"}
b: {"rules":[{"field":"f1","operator":"beginsWith","value":"b"}],"combinator":"and"}
c: {"rules":[{"field":"f1","operator":"beginsWith","value":"c"}],"combinator":"and"}
d: {"rules":[{"field":"f1","operator":"beginsWith","value":"d"}],"combinator":"and"}
e: {"rules":[{"field":"f1","operator":"beginsWith","value":"e"}],"combinator":"and"}
f: {"rules":[{"field":"f1","operator":"beginsWith","value":"f"}],"combinator":"and"}
g: {"rules":[{"field":"f1","operator":"beginsWith","value":"g"}],"combinator":"and"}
h: {"rules":[{"field":"f1","operator":"beginsWith","value":"h"}],"combinator":"and"}
i: {"rules":[{"field":"f1","operator":"beginsWith","value":"i"}],"combinator":"and"}
j: {"rules":[{"field":"f1","operator":"beginsWith","value":"j"}],"combinator":"and"}
k: {"rules":[{"field":"f1","operator":"beginsWith","value":"k"}],"combinator":"and"}
l: {"rules":[{"field":"f1","operator":"beginsWith","value":"l"}],"combinator":"and"}
m: {"rules":[{"field":"f1","operator":"beginsWith","value":"m"}],"combinator":"and"}
n: {"rules":[],"combinator":"and"}
o: {"rules":[{"field":"f1","operator":"beginsWith","value":"o"}],"combinator":"and"}
p: {"rules":[{"field":"f1","operator":"beginsWith","value":"p"}],"combinator":"and"}
q: {"rules":[{"field":"f1","operator":"beginsWith","value":"q"}],"combinator":"and"}
r: {"rules":[],"combinator":"and"}
s: {"rules":[{"field":"f1","operator":"beginsWith","value":"s"}],"combinator":"and"}
t: {"rules":[{"field":"f1","operator":"beginsWith","value":"t"}],"combinator":"and"}
u: {"rules":[{"field":"f1","operator":"beginsWith","value":"u"}],"combinator":"and"}
v: {"rules":[{"field":"f1","operator":"beginsWith","value":"v"}],"combinator":"and"}
w: {"rules":[{"field":"f1","operator":"beginsWith","value":"w"}],"combinator":"and"}
x: {"rules":[{"field":"f1","operator":"beginsWith","value":"x"}],"combinator":"and"}
y: {"rules":[{"field":"f1","operator":"beginsWith","value":"y"}],"combinator":"and"}
z: {"rules":[{"field":"f1","operator":"beginsWith","value":"z"}],"combinator":"and"}
```

I'll take a look at it today if I get a chance.

**Comment by jakeboone02** [comment]
Since it's just "r" and "n", I'm thinking this is probably an issue with newline detection (`\r` and `\n` being Carriage Return and Line Feed).

**Comment by jakeboone02** [comment]
Fixed in 6de5192b - will be included in v5.0 to be released soon.
